site/public/js/server.js

```diff
@@ -754,6 +758,8 @@ function adminTeamForm(new_team, who_id, reg_section, rot_section, user_assignme
     var param = (new_team ? 3 : members.length+2);
     members_div.append('<span style="cursor: pointer;" onclick="addTeamMemberInput(this, '+param+');" aria-label="Add More Users"><i class="fas fa-plus-square"></i> \
         Add More Users</span>');
+    if (exists_multiple_invite_member) members_div.append('<div id="multiple_invites_warning" style="color:red;font-size:small;margin-top:3px;width:75%">\
```

Comment:
Instead of adding this div through javascript let's add this div to the edit team form by default and set its display to none.
Then in this `if` statement you can toggle its visibility if the condition is reached. You can use jquery's `.toggle()` to accomplish this.

Reply:
I'm adding it to members_div, and that gets emptied at the start of this function (since it is a popup, and that changes each time it's opened again). I tried to make it its own separate div but that also messes things up when "Add More Users" is clicked.